crates/env/src/call/call_builder.rs

```diff
@@ -130,7 +130,7 @@ where
 /// ## Example 1: No Return Value
 ///
 /// The below example shows calling of a message of another contract that does
-/// not return any value back to its caller. The called function ...
+/// not return any value back to its caller. The called function…
```

Comment:
I'd leave the normal set of three periods instead of a single character. Easier to type, and imo easier to read.

If you do want to change them, I'd go with a colon instead.